src/pages/solutions/gis-strategy.astro

```diff
@@ -78,27 +81,37 @@ const page: IStandardPageMetadata = {
+      a street address, a zip code, or something that could connect it to a location or boundary? If so, it's only one
+      small step away from being useful for spatial analysis.
+    </p>
+    <p>
+      However, other data may need more work to make it spatial. Maybe you have a question that you know involves
+      "where" but it's not clear how your data can be used to answer it. We're happy to help you figure out how to make
+      your data spatial so you can answer your important questions.
     </p>
   </Section>
   <Section {...page.section[3]}>
     <Image slot="image" src={startPhoto} alt="A person jumping over a hurdle." class="w-full max-w-[400px]" />
     <p>
-      There are many ways to get started using and thinking about geospatial data. One of the best ways we’ve found for
-      agencies interested in GIS to get started is to buy an office subscription to Esri’s <ExternalLink
+      One of the best ways we've found to get started in GIS is to purchase a subscription to Esri's <ExternalLink
         href="https://www.esri.com/en-us/arcgis/products/arcgis-online/buy">ArcGIS Online platform</ExternalLink
-      >. It gives you access to all of the data, including the <a href="/documentation/sgid/">SGID on ArcGIS</a> and <ExternalLink
-        href="https://livingatlas.arcgis.com/en/home/">Esri’s Living Atlas</ExternalLink
-      >. It has tools to help you <ExternalLink href="https://www.esri.com/en-us/arcgis/products/arcgis-online/overview"
+      >. This includes access to a wide variety of data, including the <a href="/documentation/sgid/">SGID on ArcGIS</a>
+      and
+      <ExternalLink href="https://livingatlas.arcgis.com/en/home/">Esri's Living Atlas</ExternalLink>, along with tools
+      to help you <ExternalLink href="https://www.esri.com/en-us/arcgis/products/arcgis-online/overview"
         >analyze your data</ExternalLink
-      >, you can create and deploy low code and no code applications to highlight and bring attention to your GIS
-      efforts, and most importantly from UGRC’s perspective, it allows you to easily share your data with a wider
-      audience if appropriate to your business's goals and mission. ArcGIS Online can scale with your agency needs
-      without having to make a huge initial investment in GIS. Reach out to us if you have any questions about joining
-      Utah's preferred spatial data and low code no code platform.
+      >. After you've analyzed your data, you can use <ExternalLink
+        href="https://www.esri.com/en-us/arcgis/products/arcgis-experience-builder/overview"
+        >Experience Builder</ExternalLink
+      > to create and deploy low-code and no-code applications to highlight and bring attention to your results.
+    </p>
+    <p>
+      ArcGIS Online also allows you to easily share your data with a wider audience (if appropriate), including sharing
```

Comment:
or maybe the Esri share page link belongs here, instead of above

Reply:
IIUC, this sentence relates more to the sharing of the data/hosted feature services themselves for others to use in their work rather than sharing apps/maps etc.